From zbr@ioremap.net  Fri Jan 16 18:11:13 2009
From: Evgeniy Polyakov <zbr@ioremap.net>
Date: Wed, 14 Jan 2009 02:09:43 +0300
Subject: Staging: dst: kconfig update.
To: Greg KH <greg@kroah.com>
Cc: linux-kernel@vger.kernel.org, dst@ioremap.net
Message-ID: <20090113230943.GA30952@ioremap.net>
Content-Disposition: inline


On Wed, Jan 14, 2009 at 02:05:33AM +0300, Evgeniy Polyakov (zbr@ioremap.net) wrote:
> --- /dev/null
> +++ b/drivers/staging/dst/Kconfig
> @@ -0,0 +1,71 @@
> +config DST
> +	tristate "Distributed storage"
> +	depends on NET && CRYPTO && SYSFS
> +	select CONNECTOR
> +	select LIBCRC32C

Above is not needed.

Signed-off-by: Evgeniy Polyakov <zbr@ioremap.net>
Cc: Randy Dunlap <randy.dunlap@oracle.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>


---
 drivers/staging/dst/Kconfig |    4 ----
 1 file changed, 4 deletions(-)

--- a/drivers/staging/dst/Kconfig
+++ b/drivers/staging/dst/Kconfig
@@ -2,14 +2,10 @@ config DST
 	tristate "Distributed storage"
 	depends on NET && CRYPTO && SYSFS
 	select CONNECTOR
-	select LIBCRC32C
 	---help---
 	DST is a network block device storage, which can be used to organize
 	exported storages on the remote nodes into the local block device.
 
-	DST is a network block device storage, which can be used to organize
-	exported storages on the remote nodes into the local block device.
-
 	DST works on top of any network media and protocol, it is just a matter
 	of configuration utility to understand the correct addresses. The most
 	common example is TCP over IP allows to pass through firewalls and
